lib/screens/v2/widget_instance/blue_bikes.ex

```diff
@@ -35,7 +34,7 @@ defmodule Screens.V2.WidgetInstance.BlueBikes do
   @type special_station :: %{
           status: :valet | :out_of_service,
           id: String.t(),
-          arrow: Station.arrow(),
+          arrow: Arrow.t(),
```

Comment:
In the future I think it'd be nice to split changes like this into their own commit/PR if the change ends up being more than just updating the dependency — for anyone looking back at this work later, it'll be unclear why this change to the `BlueBikes` widget is included in a commit that's supposedly about elevator screens (assuming you plan to squash-merge all the work on this branch). No need to extract this particular change now, though.

Reply:
I did think about that when making this change and should have went with my gut. Will make sure to avoid this in the future.